src/fsi/console.fs

```diff
@@ -104,6 +104,123 @@ module internal Utils =
             | ' ', false -> nextWordFromIdx line (idx + 1, false)
             | _, _ -> nextWordFromIdx line (idx + 1, true)
 
+    /// An array stores ranges of full-width chars.
+    ///
+    /// Array [| a; b; c; d |] represents range [a, b] or [c, d], means chars in these ranges are full-width.
+    let private fullWidthCharRanges =
+        [|
```

Comment:
Would it be possible to make it more explicit that this is actually an array of ranges (defined by 2 characters for each range) ?

Reply:
like this ? 

``` F#
let private fullWidthCharRanges =
    Array.concat [|
        [| '\u1100'; '\u115f' |]
        [| '\u231a'; '\u231b' |]
        [| '\u2329'; '\u232a' |]
        ...
    |]
```

Reply:
Yes, although I do realize this would change the current imp based around BinarySearch.
Maybe a compromise of re-formatting the code to put each range into its own line, but keeping int a single array?

Reply:
I don't know how to change the configuration of Fantomas to make it...

Reply:
Ok, understand.